src/app/core/home/home.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { SecurityService } from '../../security/services/security.service';
+
+@Component({
+    selector: 'app-home',
+    templateUrl: './home.component.html',
+    styleUrls: ['./home.component.scss']
+})
+export class HomeComponent implements OnInit {
+
+    constructor(private security: SecurityService, private http: HttpClient) {
+    }
+
+    ngOnInit() {
+    }
+
+    isInRole(roleType: string) {
```

Comment:
would that mean we need such method in all components then, could not that be more generic somehow, in app or somewhere?

Reply:
Currently, it's used just in this one place and its wrapper on a more generic hasRole(x) of a User class.

Reply:
I'd say that's fine for now. We can improve it later on by creating ngrx'ish State for that